authz/rbac_translator.go

```diff
@@ -94,7 +94,8 @@ func getStringMatcher(value string) *v3matcherpb.StringMatcher {
 	switch {
 	case value == "*":
 		return &v3matcherpb.StringMatcher{
-			MatchPattern: &v3matcherpb.StringMatcher_SafeRegex{},
+			MatchPattern: &v3matcherpb.StringMatcher_SafeRegex{
+				SafeRegex: &v3matcherpb.RegexMatcher{Regex: "^\\S+$"}},
```

Comment:
Is this explicitly excluding strings with whitespace in them on purpose?  From your earlier comment it sounded like you wanted "non-empty strings".

Reply:
Right! Updated regex to allow whitespace characters.